static/src/javascripts/es6/projects/common/modules/crosswords/clues.js

```diff
@@ -44,7 +45,8 @@ export default class Clues extends React.Component {
             .map((clue) =>
                 <Clue
                     key={clue.entry.id}
-                    number={clue.entry.number + '.'}
+                    number={clue.entry.number}
+                    displayNumber={clue.entry.humanNumber}
```

Comment:
Perhaps we should stay consistent with our terminology here. `humanNumber` works fine?